proxy/mux/mux_test.go

```diff
@@ -452,26 +457,33 @@ func (s *ServerSuite) TestSNI(c *C) {
 	b := MakeBatch(Batch{
 		Host:     "localhost",
 		Addr:     "localhost:41000",
-		Route:    `Host("localhost") && Path("/")`,
+		Route:    `Path("/path1")`,
 		URL:      e.URL,
 		Protocol: engine.HTTPS,
 		KeyPair:  &engine.KeyPair{Key: localhostKey, Cert: localhostCert},
 	})
 	b2 := MakeBatch(Batch{
 		Host:     "otherhost",
 		Addr:     "localhost:41000",
-		Route:    `Host("otherhost") && Path("/")`,
+		Route:    `Path("/path2")`,
 		URL:      e2.URL,
 		Protocol: engine.HTTPS,
-		KeyPair:  &engine.KeyPair{Key: localhostKey2, Cert: localhostCert2},
+		KeyPair:  &engine.KeyPair{Key: otherHostKey, Cert: otherHostCert},
 	})
 	b2.H.Settings.Default = true
 	c.Assert(s.mux.Init(MakeSnapshot(b, b2)), IsNil)
 	c.Assert(s.mux.Start(), IsNil)
 
-	// When/Then
-	c.Assert(GETResponse(c, b.FrontendURL("/"), testutils.Host("localhost")), Equals, "Hi, I'm endpoint 1")
-	c.Assert(GETResponse(c, b.FrontendURL("/"), testutils.Host("otherhost")), Equals, "Hi, I'm endpoint 2")
+	// For the same host, but different paths, return different endpoints (routing test)
```

Comment:
No need to test routing here, it is tested in other tests, and just clutters this one.